core/src/test/java/org/ehcache/EhcacheBasicGetTest.java

```diff
@@ -148,7 +150,8 @@ public void testGetNoStoreEntryCacheLoadingException() throws Exception {
     try {
       ehcache.get("key");
       fail();
-    } catch (CacheLoadingException e) {
+    } catch (CachePassThroughException e) {
```

Comment:
Looking a bit more at the code, that probably means doing some work on the `FakeStore` used here to match what happened in the real stores.